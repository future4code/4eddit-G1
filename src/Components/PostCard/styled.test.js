import React from "react";
import renderer from "react-test-renderer";
import { PostCardMainContainer, HeaderPostContent, UserName, BodyPostContent, PostTitle, FooterPostContent, ButtonArea, ComentWords, UpArrow, DownArrow } from './styled'

describe("Styleds", () => {
    // Mesma coisa do que test()
    it("PostCardMainContainer", () => {
      const tree = renderer.create(<PostCardMainContainer />).toJSON();
      expect(tree).toMatchSnapshot();
    });
    // Mesma coisa do que test()
    it("DownArrow", () => {
      const tree = renderer.create(<DownArrow />).toJSON();
      expect(tree).toMatchSnapshot();
    });
    // Mesma coisa do que test()
    it("ButtonArea", () => {
      const tree = renderer.create(<ButtonArea />).toJSON();
      expect(tree).toMatchSnapshot();
    });
    // Mesma coisa do que test()
    it("ComentWords", () => {
      const tree = renderer.create(<ComentWords />).toJSON();
      expect(tree).toMatchSnapshot();
    });
    // Mesma coisa do que test()
    it("UpArrow", () => {
      const tree = renderer.create(<UpArrow />).toJSON();
      expect(tree).toMatchSnapshot();
    });
    // Mesma coisa do que test()
    it("HeaderPostContent", () => {
      const tree = renderer.create(<HeaderPostContent />).toJSON();
      expect(tree).toMatchSnapshot();
    });
    // Mesma coisa do que test()
    it("UserName", () => {
      const tree = renderer.create(<UserName />).toJSON();
      expect(tree).toMatchSnapshot();
    });
    // Mesma coisa do que test()
    it("BodyPostContent", () => {
      const tree = renderer.create(<BodyPostContent />).toJSON();
      expect(tree).toMatchSnapshot();
    });
    // Mesma coisa do que test()
    it("FooterPostContent", () => {
      const tree = renderer.create(<FooterPostContent />).toJSON();
      expect(tree).toMatchSnapshot();
    });
    // Mesma coisa do que test()
    it("PostTitle", () => {
      const tree = renderer.create(<PostTitle />).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });