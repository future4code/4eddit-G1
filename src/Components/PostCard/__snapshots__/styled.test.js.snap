// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Styleds BodyPostContent 1`] = `
<div
  className="sc-ifAKCX Apgsu"
/>
`;

exports[`Styleds ButtonArea 1`] = `
<div
  className="sc-bxivhb hmEHNs"
/>
`;

exports[`Styleds ComentWords 1`] = `
<span
  className="sc-gzVnrw dOAkBY"
/>
`;

exports[`Styleds DownArrow 1`] = `
<img
  className="sc-dnqmqq imYJvK"
/>
`;

exports[`Styleds FooterPostContent 1`] = `
<div
  className="sc-EHOje liqMYS"
/>
`;

exports[`Styleds HeaderPostContent 1`] = `
<div
  className="sc-htpNat dLhcg"
/>
`;

exports[`Styleds PostCardMainContainer 1`] = `
<div
  className="sc-bdVaJa jZSpap"
/>
`;

exports[`Styleds PostTitle 1`] = `
<h3
  className="sc-bwzfXH ivdGIV"
/>
`;

exports[`Styleds UpArrow 1`] = `
<img
  className="sc-htoDjs eygCcn"
/>
`;

exports[`Styleds UserName 1`] = `
<span
  className="sc-bZQynM bMyLxC"
/>
`;
